sale_elaboration/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module extends the functionality of sale orders to allow add an
+elaboration in a sale order line and add an extra line with an elaboration
+product linked to the elaboration.
```

Comment:
Define what is an elaboration: `An elaboration is a process that needs to be done over the product, usually on picking/handling phase. It doesn't modify too much the product for needing an specific product, but it adds a surcharge on the final price`.